test/ussd_popi_faq.test.js

```diff
@@ -0,0 +1,597 @@
+var vumigo = require("vumigo_v02");
+var AppTester = vumigo.AppTester;
+var assert = require('assert');
+var fixtures_IdentityStore = require('./fixtures_identity_store');
+var fixtures_StageBasedMessaging = require('./fixtures_stage_based_messaging');
+var fixtures_MessageSender = require('./fixtures_message_sender');
+var fixtures_Hub = require('./fixtures_hub');
+var fixtures_Jembi = require('./fixtures_jembi');
+var fixtures_ServiceRating = require('./fixtures_service_rating');
+
+var utils = require('seed-jsbox-utils').utils;
+
+describe("app", function() {
+    describe("for ussd_popi_faq use", function() {
+        var app;
+        var tester;
+
+        beforeEach(function() {
+            app = new go.app.GoNDOH();
+
+            tester = new AppTester(app);
+
+            tester
+                .setup.config.app({
+                    name: 'ussd_popi_faq',
+                    env: 'test',
+                    metric_store: 'test_metric_store',
+                    testing_today: "2014-04-04 07:07:07",
+                    testing_message_id: '0170b7bb-978e-4b8a-35d2-662af5b6daee',
+                    logging: "off",
+                    no_timeout_redirects: [
+                        "state_start", "state_end_not_pregnant", "state_end_consent_refused",
+                        "state_end_success", "state_registered_full", "state_registered_not_full",
+                        "state_end_compliment", "state_end_complaint", "state_end_go_clinic"],
```

Comment:
You can remove a bunch of states from here since they don't exist in this app. This is a list of states that shouldn't redirect to the `state_timed_out` if a gets disconnected and redials. I think you'll only need `start_state` here.